app/models/ability.rb

```diff
@@ -8,11 +8,15 @@ def initialize(user)
     if user.role? :admin
       can :read, Question
       can :create, Question
+      can :manage, Question
+      can :manage, Answer
```

Comment:
Do you need this line? We're not creating Answers directly (they're being generated as part of a Question create) so if you remove this does everything work as expected still? (See the comment on `answers_controller.rb`)